subworkflows/local/sourmash.nf

```diff
@@ -4,14 +4,15 @@
 
 include { SOURMASH_GATHER                   } from '../../modules/nf-core/sourmash/gather/main'
 include { SOURMASH_SKETCH as GENOMES_SKETCH } from '../../modules/nf-core/sourmash/sketch/main'
+include { SOURMASH_INDEX as GENOMES_INDEX   } from '../../modules/nf-core/sourmash/index/main'
 include { SOURMASH_SKETCH as SAMPLES_SKETCH } from '../../modules/nf-core/sourmash/sketch/main'
 
 workflow SOURMASH {
     take:
-        reference_genomes
         samples_reads
         indexes
-        genomeinfo
+        ch_user_genomeinfo
+        ncbi_genomeinfo_files
```

Comment:
In the name of consistency, all parameters that are channels should start with `ch_`. I'm not changing here, because I might forget something further down.